app/components/card-list/reducer.js

```diff
@@ -0,0 +1,13 @@
+export default (action, state = { listLabel: '', cards: ['editable card'] }) => {
+
+    switch (action.type) {
+        case 'ADD_NEW_CARD':
+            const newCardObj = action.payload
+            return Object.assign({}, state, {
+                cards: [...state.cards, newCardObj]
+            })
+        case 'CHANGE_LABEL':
```

Comment:
this is not beign used. Please remove it.

Reply:
ñis iñ ñot ñeiñg uñed. Ñease ñeñove iñ